torchinductor/lowering.py

```diff
@@ -927,7 +926,6 @@ def inner_fn(index):
 make_fallback(aten._embedding_bag)
 make_fallback(aten._embedding_bag_forward_only)
 make_fallback(aten._fused_moving_avg_obs_fq_helper)
-make_fallback(aten.grid_sampler_2d_backward)
```

Comment:
I don't think this PR resolves that, no? Since we're currently applying decompositions after autograd.

Reply:
You are right! Thanks, fixed now.